src/Tizen.NUI.Components/Controls/RecyclerView/Layouter/LinearLayouter.cs

```diff
@@ -204,10 +204,19 @@ public override void Initialize(RecyclerView view)
             bool failed = false;
 
             //Final Check of FirstIndex
-            while (colView.InternalItemSource.IsHeader(firstIndex) ||
+
+            if (colView.InternalItemSource.Count - 1 < firstIndex)
```

Comment:
how about changing the condition to be the same with the condition in the while loop?
```
if (colView.InternalItemSource.Count - 1 <= firstIndex)
```